### PR TITLE
fix(signup): make email suggestion tooltip keyboard accessible

### LINKED ISSUE #2185: email suggestion tooltip is not keyboard accessible
This little fella has no way of tabbing into it.

screen shot 2015-03-04 at 13 42 48

Ref #2009

### PR DESCRIPTION
**Pull request by riadhchtara**
Fixes #2185

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@vladikoff - could you review this one?

**Comment by vladikoff** [comment]
Could we improve the highlight / contrast for the selected item: 
image

It's hard to see the selected action.

File: app/scripts/lib/mailcheck.js [hunk 1]
```diff
@@ -69,14 +69,20 @@ function (Tooltip, Url) {
...
+              // the user has used the suggestion
+              metrics.logEvent('tooltip.mailcheck-used');
+              tooltip._destroy();
+            }
           });
         }
       });
     }
+
+    element[0].previousValue =  element.val();
```
Comment (vladikoff) [inline app/scripts/lib/mailcheck.js]: Could we reinforce this and add a check if `var element = $(target);` found an element in `element[0]`

**Comment by vladikoff** [comment]
Need a unit test for the tab index, see https://github.com/mozilla/fxa-content-server/blob/master/app/tests/spec/views/sign_up.js#L647

**Comment by vladikoff** [comment]
This is mostly ready, just need to address the 3 notes above ^. Thanks for sending this PR!

File: app/scripts/lib/mailcheck.js (hunk 1 above)
Comment (riadhchtara) [inline app/scripts/lib/mailcheck.js]: done

**Comment by riadhchtara** [comment]
hi @vladikoff:
thanks for the review.
Could you please do another quick check?
thanks :)

File: app/scripts/lib/mailcheck.js [hunk 2]
```diff
@@ -69,14 +69,21 @@ function (Tooltip, Url) {
...
+              element.val(suggestion.full);
+              // the user has used the suggestion
+              metrics.logEvent('tooltip.mailcheck-used');
+              tooltip._destroy();
+            }
           });
         }
       });
     }
+    if (element.length) {
```
Comment (vladikoff) [inline app/scripts/lib/mailcheck.js]: Instead could we `return` right after `var element = $(target);` if it cannot be found

Comment (riadhchtara) [inline app/scripts/lib/mailcheck.js]: OK

**Comment by vladikoff** [comment]
@ryanfeeley or @johngruen could you suggest a style for TAB highlight for popups.
Currently this PR makes them #000.
image

**Comment by ryanfeeley** [comment]
How does text color white, 1 pixel white dotted border look?

**Comment by vladikoff** [comment]
@ryanfeeley 
image

**Comment by ryanfeeley** [comment]
Looks awesome. The domain and close aren't focused at the same time are they?

**Comment by vladikoff** [comment]
@ryanfeeley no, just for the screenshot purposes 

@riadhchtara ^

**Comment by riadhchtara** [comment]
thanks for the review :)
I updated the style of the domain and the close :)

**Comment by vladikoff** [comment]
Thanks @riadhchtara !

**Comment by riadhchtara** [comment]
thanks :-)

**Pull request merged**